### ISSUE TITLE
osc co -o outdir Project leaves packages in unknown state

### ISSUE THREAD

**Issue by kaltsi** [openSUSE/osc#211]
```
$ osc co -o outdir Projectname
$ cd outdir
$ osc status
```
All packages are shown as '?'.

Also unable to do:

```
$ osc co Projectname
$ osc co -o outdir2 Projectname

osc: project 'Projectname' already exists.
```

**Comment by kaltsi** [comment]
Pull request #212 fixes both cases.

**Comment by marcus-h** [comment]
Fixed by PR#212 (see commit c4e4064778f0dc2f0c0f7e7d172c72bcaca253f8).
Thanks!
